### PR TITLE
Update whats-new-in-defender-for-office-365.md

### PR DESCRIPTION
**Pull request by ajaj-shaikh**
Added URL click alerts enhancement as part of Oct'22 updates

### CODE REVIEW

File: microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ For more information on what's new with other Microsoft Defender security produc
 - **[Manage your allows and blocks in the Tenant Allow/Block List](manage-tenant-allow-block-list.md):**
   - With **allow expiry management** (currently in private preview), if Microsoft has not learned from the allow, Microsoft will automatically extend the expiry time of allows, which are going to expire soon, by 30 days to prevent legitimate email from going to junk or quarantine again.
   - Customers in the government cloud environments will now be able to create allow and block entries for URLs and attachments in the Tenant Allow/Block List using the admin URL and email attachment submissions. The data submitted through the submissions experience will not leave the customer tenant, thus satisfying the data residency commitments for government cloud clients.
+- **[Enhancement in URL click alerts]**
+  - With the new lookback scenario, "A potentially malicious URL click was detected" alert will now include any clicks in the **past 48 hours** (for emails) from the time the malicious URL verdict is identified.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md]:
```suggestion
- **Enhancement in URL click alerts:**
  - With the new lookback scenario, the "A potentially malicious URL click was detected" alert will now include any clicks during the _past 48 hours_ (for emails) from the time the malicious URL verdict is identified. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ajaj-shaikh, kindly commit the suggested changes. Many thanks.

**Comment by ajaj-shaikh** [comment]
@MSFTTracyP can you please merge this PR?

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by JohanFreelancer9** [comment]
@yogkumgit - my suggested changes have not been implemented yet.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Pull request merged**